nmtwizard/preprocess/tu.py

```diff
@@ -210,43 +210,71 @@ def __init__(self,
                  alignment=None,
                  source_tokenizer=None,
                  target_tokenizer=None):
-        self.__source = TranslationSide(source, tokenizer=source_tokenizer)
+        self.__source = [TranslationSide(source, tokenizer=source_tokenizer)]
         self.__target = None
         self.__metadata = metadata if metadata is not None else [None] #TODO: proper metadata
         self.__annotations = annotations
         self.__alignment = None
 
         if target is not None:
-            self.__target = TranslationSide(target, tokenizer=target_tokenizer)
+            self.__target = [TranslationSide(target, tokenizer=target_tokenizer)]
             if alignment is not None:
                 self.__alignment = Alignment(alignments=alignment)
 
+    def add_source(self, source, source_tokenizer=None):
+        self.__source.append(TranslationSide(source, tokenizer=source_tokenizer))
```

Comment:
It is implied that the tokenizer is for the source, so we can simplify the argument name:

```suggestion
    def add_source(self, source, tokenizer=None):
        self.__source.append(TranslationSide(source, tokenizer=tokenizer))
```